nucypher/characters/control/specifications/enrico.py

```diff
@@ -16,25 +16,52 @@
 """
 
 import click
+from marshmallow import post_load
 
-from nucypher.characters.control.specifications import fields
-from nucypher.characters.control.specifications.base import BaseSchema
+from nucypher.characters.control.specifications import fields, exceptions
 from nucypher.cli import options
+from nucypher.cli.types import EXISTING_READABLE_FILE
+from nucypher.characters.control.specifications.base import BaseSchema
 
 
 class EncryptMessage(BaseSchema):
 
     # input
     message = fields.Cleartext(
-        required=True, load_only=True,
+        load_only=True,
+        allow_none=True,
         click=click.option('--message', help="A unicode message to encrypt for a policy")
     )
 
+    filepath = fields.FileField(
+        load_only=True,
+        allow_none=True,
+        click=click.option('--file', help="Filepath to plaintext file to encrypt", type=EXISTING_READABLE_FILE)
+    )
+
     policy_encrypting_key = fields.Key(
         required=False,
         load_only=True,
-        click=options.option_policy_encrypting_key())
+        click=options.option_policy_encrypting_key()
+    )
+
+    @post_load()
+    def format_method_arguments(self, data, **kwargs):
+        """
+        input can be through either the file input or a raw message,
+        we output one of them as the "plaintext" arg to enrico.encrypt_message
+        """
+
+        if data.get('message') and data.get('filepath'):
+            raise exceptions.InvalidArgumentCombo("choose either a message or a filepath but not both.")
+
+        if data.get('message'):
+            data = bytes(data['message'], encoding='utf-8')
+        else:
+            data = data['filepath']
```

Comment:
I assume that this is the file data and not the string path of the file...?

Reply:
yes at that point, the filepath field's serialization method has read the data from the file.

Reply:
I figured, it's just that the key `filepath` through me off.

Reply:
oh - it's not really the filepath?

Reply:
That method basically allows you to customize the output of the serializer.  It is run after all of the fields process the input data in their own ways.  The filepath field deserializes its input by reading the file data so that’s what is present in that key right there.

Reply:
Minor nit pick but is it possible to just change the key to be `file` instead of `filepath`? Just to make it clearer.

Reply:
@KPrasch you cool with that?  I am.  Lets do it.

Reply:
I agree. `filepath` is confusing.

Reply:
:heavy_check_mark: